src/rdb.c

```diff
@@ -1840,6 +1841,18 @@ static int _lpEntryValidation(unsigned char *p, unsigned int head_count, void *u
         }
     }
 
+    /* Validate TTL field. */
+    if (data->count % data->tuple_len == 2) {
```

Comment:
Shouldn't `tuple_len` be 3 for fields with TTL?

Reply:
`(data->count)++;` is after this code, so it's TTL field when count is 2.